sqlmesh/core/engine_adapter/base.py

```diff
@@ -87,6 +87,7 @@ class EngineAdapter:
     COMMENT_CREATION_VIEW = CommentCreationView.IN_SCHEMA_DEF_AND_COMMANDS
     MAX_TABLE_COMMENT_LENGTH: t.Optional[int] = None
     MAX_COLUMN_COMMENT_LENGTH: t.Optional[int] = None
+    ESCAPE_COMMENT_BACKSLASH = True
```

Comment:
What's the reasoning behind only switching this off for postgres?

Reply:
Postgres does the escaping itself - if we also do it the backslashes are doubled in the registered comment

Reply:
Ah, is this only specific to table / column comments? I was wondering if this is something that can be controlled by `dialect.tokenizer_class.STRING_ESCAPES` or some other constant available through SQLGlot.

Reply:
Yep, just for table/column comments